### PR TITLE
About me page

### PR DESCRIPTION
**Pull request by PhoenixMP**
First-draft of the "about" page, copying the format of the original website page. 

image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,8 +38,14 @@
     },
     "dependencies": {
         "@formkit/auto-animate": "^0.8.1",
+        "@fortawesome/fontawesome-free": "^6.5.1",
```
Comment (RyanDrx) [inline package.json:41]: Probably do not need this dependency, as the nuxt-icon and hero-icons takes care of it.  Also pretty sure fontawesome is already installed and the icons can be used. 

We can use any icon from here https://icones.js.org/ which includes every icon library


example usage


```
  <icon name="bxs:quote-right" size="1em" class="flex-none min-w-8"></icon>
```

**Review by RyanDrx — CHANGES_REQUESTED** [review]
Some minor nits or otherwise some advice. 

Everything looks excellent, and you did a great job emulating the current page.  I like how you destructured the components into layout and data. That is really helpful for making changes later or for quickly spinning up a page for SEO targeting. 

I also viewed with mobile and it looks great. The only issue is Erin's portrait is HUGE on mobile. :)

File: components/AboutMePage/Qualification.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const props = defineProps({ data: String });
+
+</script>
+
+<template>
+  <section class="mx-4">
+    <div class="flex items-center justify-center">
+      <font-awesome-icon :icon="data.icon" class="mr-4 text-5xl text-pink-900 text-opacity-50" />
+    </div>
+    <div class="mt-5 mb-5 lg:text-4xl lg:mb-5 text-2xl font-medium">
```
[the remark this answers is not in the document]
Comment (PhoenixMP) [reply components/AboutMePage/Qualification.vue:15]: I like it better center:
image

File: pages/about.vue [hunk 3]
```diff
@@ -1,19 +1,126 @@
...
+    title: "Life Outside My Practice",
+    paragraphs: [
+      {
+        text: "I share my current residence in Seattle, WA with my husband and my favorite therapist: a soul-reading, sun-soaking, smiling Shiba Inu named Kota. ",
+      },
+      {
+        text: "In my free time, I enjoy cooking, baking, and eating (while jamming to some blues, oldies, and Dolly Parton) as well as being outdoors and taking in the splendor of the Pacific Northwest. I am a yogi at heart and enjoy playing virtually any sport that involves a ball. ",
+      },
+      {
+        text: "I am eager to help driven, smart, talented individuals embrace their strengths and start a career and life they aspire to.",
```
[the remark this answers is not in the document]
Comment (PhoenixMP) [reply pages/about.vue:89]: I saw you doing an array of strings on a different component and that's where I got the idea, but I noticed that some sections on the about page had altered text in the paragraphs. Those text objects have an added key of "styleType" (fig-1). Right now in my AboutMePage-Section component, an extra class for styling is applied to the paragraph text if that styleType key exists in the object (fig-2). 

I could have added styleType to all  of the objects for clarity and set those values to "" if there's no extra styling needed, and change the component to search for a styleType value >0?
Or I could have a separate styleStype array and define if there are additional styleTypes in the corresponding index.

Let me know if there's a better way you see that I am not thinking of!

image
image

File: components/AboutMePage/Intro.vue [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import { defineProps } from "vue";
+
+// Use 'defineProps' to access the props
+const props = defineProps({ data: Object });
```
[the remark this answers is not in the document]
Comment (PhoenixMP) [reply components/AboutMePage/Intro.vue:5]: I've declared more specific types for each of the components in the AboutMePage:

image

File: pages/about.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (RyanDrx) [reply pages/about.vue]: Ah ok, that's makes sense. .

File: components/AboutMePage/Intro.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (RyanDrx) [reply components/AboutMePage/Intro.vue]: Thanks looks great

File: package.json (hunk 1 above)
Comment (PhoenixMP) [reply package.json:41]: Whoops, I thought I responded to this yesterday too. 

I used the <Icon> component and found similar icons:
image

**Review by RyanDrx — APPROVED** [review]

**Pull request merged**